fre/app/generate_time_averages/cdoTimeAverager.py

```diff
@@ -9,7 +9,17 @@ class inheriting from abstract base class timeAverager
     '''
 
     def generate_timavg(self, infile=None, outfile=None):
-        ''' use cdo package routines via python bindings '''
+        """
+        use cdo package routines via python bindings        
+
+        :param self: 
+        :param infile: path to history file, or list of paths
+        :type infile: str, list
+        :param outfile: path to where output file should be stored
+        :type outfile: str
+        :param self: time scale for climatology. Accepted variables vary based on pkg ('all','seas','month') 
```

Comment:
I'm not sure this object attribute should be in the docstring, but it does look useful.